modules/ROOT/pages/tableofcontents.adoc

```diff
@@ -3,10 +3,15 @@
 :description: Insert a simple Table of Contents into the {productname} editor
 :keywords: tableofcontents, tableofcontents_depth, tableofcontents_class, tableofcontents_header, toc
 :pluginname: Table of Contents
+:pluginnamelower: table of contents
 :plugincode: tableofcontents
-:pluginminimumplan: tierone
+:pluginminimumplan: tiertwo
 
-The `+tableofcontents+` plugin will generate basic _Table of Contents_ and insert it into the editor at the current cursor position. Items for the table will be taken from the headers found in the content.
+include::partial$misc/admon-premium-plugin.adoc[]
+
+The {pluginname} plugin will generate basic {pluginnamelower} and insert it into the editor at the current cursor position. Items for the table will be taken from the headers found in the content.
```

Comment:
```suggestion
The {pluginname} plugin generates a basic {pluginnamelower} (ToC) and inserts it into the editor at the current cursor position. ToC entries are generated from header elements (`h1 – h7` elements) in the content.
```

Minor edits. I resisted the temptation to `s/cursor position/insertion point/`. The I-beam pointer is not a cursor. But that’s an editorial change that I need to argue for and then do across the entire documentation suite.

Reply:
`h7` doesn't exist... I'm guessing that was a typo and was meant to be `h6`?